src/main/java/com/joyent/manta/monitor/JettyServerBuilderModule.java

```diff
@@ -30,6 +27,6 @@ public void configure(final Binder binder) {
                 .addPort(jettyServerPort)
                 .build();
 
-        binder.install(new JerseyModule(jerseyConfig));
+        binder.install(new MantaMonitorJerseyModule(jerseyConfig));
```

Comment:
Separate instantiation of `MantaMonitorJerseyModule` to make the stacktrace easier to understand.